### ISSUE TITLE
About the theano version

### ISSUE THREAD

**Issue by lixin4ever** [happywwy/Coupled-Multi-layer-Attentions#1]
Hi Wenya,
    Thanks for releasing the code.  It seems that the code can not run normally and the problem lies in the recurrence function of GRU (i.e., the function "gru"). Can you tell me the theano version you use in the experiment?

**Comment by ArrowLuo** [comment]
@lixin4ever I tried several versions of Theano, but all of them didn't work well. At last, Theano==0.8.0 with CPU as a compromise proposal can get the results. FYI. However, GPU couldn't work caused by the version CUDnn, I think.

**Comment by lixin4ever** [comment]
@ArrowLuo  Thanks for your notification. Actually, I have figured it out by changing some part of the source code.

**Comment by hnstr** [comment]
Has anyone made it work with GPU support yet?

**Comment by JamesRiddle** [comment]
@lixin4ever I have tried this code with several versions of Theano, but I am unable to run it successfully. Can you help me out to run this code by providing your edited source code?
Thanks in anticipation

**Comment by hnstr** [comment]
@JamesRiddle  I made it run with python2 theano 0.8.2. If that doesn't work for you, let me know.

**Comment by lixin4ever** [comment]
@JamesRiddle I am ok for providing the modified code but I think you can follow the idea provided by @hnstr first. If that doesn't work, we can discuss here so that the following users can know what is wrong with the original code.

**Comment by happywwy** [comment]
Yes, the code should be running on theano 0.8.2. That's the version I used
for the implementation.

**Comment by zhaohe1995** [comment]
> @JamesRiddle I am ok for providing the modified code but I think you can follow the idea provided by @hnstr first. If that doesn't work, we can discuss here so that the following users can know what is wrong with the original code.

Hi, lixin. Can you share your modified code here? I also failed to run the original code, even under the python2+theano0.8.2.  It will be very kind of you to let us know what is wrong with the original code.
